cmd/http/server.go

```diff
@@ -211,7 +205,7 @@ func NewServer(addrs []string, handler http.Handler, getCert certs.GetCertificat
 			CipherSuites:             defaultCipherSuites,
 			CurvePreferences:         secureCurves,
 			MinVersion:               tls.VersionTLS12,
-			NextProtos:               []string{"h2", "http/1.1"},
+			NextProtos:               []string{"http/1.1", "h2"},
```

Comment:
Could you update the commit message that with this change we are making http1.1 the protocol for internal HTTP/RPC communication while the http2 protocol is still available for (external) clients that demand it?

Reply:
Done